cpp/test/IceStorm/rep1/Sub.cpp

```diff
@@ -55,6 +55,9 @@ Sub::run(int argc, char** argv)
         throw invalid_argument(os.str());
     }
 
+    // With "mx", this creates the Ice.Admin object adapter.
+    auto adapter = communicator->createObjectAdapterWithEndpoints("SingleAdapter", "default");
```

Comment:
Moved here because one test (most likely by accident) relies on a "Xxx ready" getting printed before failing to connect to the topic manager. Failing before "Xxx ready" => test fails.